Mathlib/Analysis/InnerProductSpace/Symmetric.lean

```diff
@@ -74,17 +74,48 @@ theorem IsSymmetric.apply_clm {T : E →L[𝕜] E} (hT : IsSymmetric (T : E →
     ⟪T x, y⟫ = ⟪x, T y⟫ :=
   hT x y
 
+@[aesop safe apply]
 theorem isSymmetric_zero : (0 : E →ₗ[𝕜] E).IsSymmetric := fun x y =>
   (inner_zero_right x : ⟪x, 0⟫ = 0).symm ▸ (inner_zero_left y : ⟪0, y⟫ = 0)
 
+@[aesop safe apply]
 theorem isSymmetric_id : (LinearMap.id : E →ₗ[𝕜] E).IsSymmetric := fun _ _ => rfl
 
-theorem IsSymmetric.add {T S : E →ₗ[𝕜] E} (hT : T.IsSymmetric) (hS : S.IsSymmetric) :
+@[aesop safe apply]
+theorem isSymmetric.add {T S : E →ₗ[𝕜] E} (hT : T.IsSymmetric) (hS : S.IsSymmetric) :
     (T + S).IsSymmetric := by
   intro x y
   rw [LinearMap.add_apply, inner_add_left, hT x y, hS x y, ← inner_add_right]
   rfl
 
+@[aesop safe apply]
+theorem isSymmetric.sub {T S : E →ₗ[𝕜] E} (hT : T.IsSymmetric) (hS : S.IsSymmetric) :
+    (T - S).IsSymmetric := by
+  intro x y
+  rw [LinearMap.sub_apply, inner_sub_left, hT x y, hS x y, ← inner_sub_right]
+  rfl
+
+@[aesop safe apply]
+theorem isSymmetric.smul {T : E →ₗ[𝕜] E} (hT : T.IsSymmetric) {c : 𝕜} (hc : conj c = c) :
+    c • T |>.IsSymmetric := by
+  intro x y
+  simp only [smul_apply, inner_smul_left, hc, hT x y, inner_smul_right]
+
+@[aesop safe apply]
+lemma isSymmetric.mul_of_comm {T S : E →ₗ[𝕜] E} (hT : T.IsSymmetric) (hS : S.IsSymmetric)
+    (hST : Commute S T) : (S * T).IsSymmetric := by
+  refine fun x y ↦ ?_
+  nth_rw 1 [hST]
+  simp only [mul_apply]
+  rw [← hS, hT]
```

Comment:
```suggestion
  fun _ _ ↦ by rw [mul_apply, hS, hT, hST, mul_apply]
```

Reply:
AWESOME! I was trying to do something like this.